README.md

```diff
@@ -120,14 +121,14 @@ The following devices are supported by this version of liquidctl.  See each guid
 | Power supply | [Corsair RM650i, RM750i, RM850i, RM1000i](docs/corsair-hxi-rmi-psu-guide.md) | USB HID | |
 | Power supply | [NZXT E500, E650, E850](docs/nzxt-e-series-psu-guide.md) | USB HID | <sup>_p_</sup> |
 
-<sup>_L_</sup> _Requires the `--legacy-690lc` flag._  
-<sup>_U_</sup> _Requires `--unsafe` features._  
-<sup>_Z_</sup> _Requires replacing the device driver [on Windows](#installing-on-windows)._  
-<sup>_a_</sup> _Architecture-specific limitations._  
-<sup>_e_</sup> _Experimental support._  
-<sup>_n_</sup> _New driver, only available on git._  
-<sup>_p_</sup> _Only partially supported._  
-<sup>_x_</sup> _Only supported on Linux._  
```

Comment:
The trailing whitespace is semantically relevant here, and required by Markdown to force a line break.